fixtures/demo.json

```diff
@@ -1122,6 +1122,7 @@
       "collection": 1,
       "title": "mackerel",
       "file": "original_images/mackerel.webp",
+      "description": "A school of Pacific Mackerel swimming in dark waters.",
```

Comment:
This one seems to be from https://cma.recreation.parks.lacity.gov/marine-life/southern-california-species/fish/pacific-mackerel. Again we should probably replace this.

Reply:
I discovered today that there actually are NOT more mackerel in Kent than people in China. Not the biggest priority but something to be aware of: https://media.mcsuk.org/documents/Mackerel_no_longer_sustainable_choice_says_environmental_charity.pdf